src/nvim/ex_getln.c

```diff
@@ -2263,20 +2280,56 @@ static void draw_cmdline(int start, int len)
     msg_outtrans_len(ccline.cmdbuff + start, len);
 }
 
+void ui_ext_cmdline_char(int c, int shift)
```

Comment:
Why? `"cmdline_char"` (or `ui_event("cmdline_char"` to be even preciser) is already greppable.